cmd/gen-manifests/main.go

```diff
@@ -602,14 +612,15 @@ func u(s string) string {
 
 func main() {
 	// common args
-	var outputDir, cacheRoot, configPath string
+	var outputDir, cacheRoot, configPath, imgConfigPath string
 	var nWorkers int
 	var metadata, skipNoconfig, skipNorepos bool
 	flag.StringVar(&outputDir, "output", "test/data/manifests/", "manifest store directory")
 	flag.IntVar(&nWorkers, "workers", 16, "number of workers to run concurrently")
 	flag.StringVar(&cacheRoot, "cache", "/tmp/rpmmd", "rpm metadata cache directory")
 	flag.BoolVar(&metadata, "metadata", true, "store metadata in the file")
 	flag.StringVar(&configPath, "config", "test/config-map.json", "configuration file mapping image types to configs")
+	flag.StringVar(&imgConfigPath, "image-config", "", "image config file to use for all images (overrides -config)")
```

Comment:
Let's rename `-config` to `-config-map` and use `-config` for the new one.  I think that would make the flag names clearer.  And it's a development tool so we don't really care about backwards compatibility.

Reply:
I didn't want to break backward compatibility, but I agree with your suggestion :wink:

Reply:
Done (hopefully I didn't miss any place where to change the option name :innocent: ).